client/src/App.js

```diff
@@ -29,7 +25,9 @@ const App = () => {
           <Route path="/">
             <Explore />
           </Route>
-
+          <Route path="/social-tag-test">
```

Comment:
i think the "/" route should go last-- super nit but I had to switch the order to get the router to work.